tests/api/permissions/test_app_permissions.py

```diff
@@ -96,5 +97,7 @@ def test_authenticated_user_has_basic_perms(client, users):
 def test_permission(client, app, users, view, user, expected_status):
     u = users[user]
     client.force_login(u)
-    response = view(client, app)
-    assert response.status_code == expected_status
+
+    with patch("controlpanel.api.views.models.App.delete") as _:
```

Comment:
I believe you don't need the `as _` here, you can just leave it off. Some linters / code quality checkers might complain about an unused object (while others might understand the `_` convention).

Reply:
Cool, thanks for the tip, I'll give it a try and see if it works 👍

Reply:
TIL!